lib/readline.js

```diff
@@ -24,7 +24,7 @@ exports.createInterface = function(input, output, completer, terminal) {
 };
 
 
-function Interface(input, output, completer, terminal) {
+function Interface(input, output, completer, terminal, historySize) {
```

Comment:
Well, you are adding it here. Why not take the `historySize` from the options object, and otherwise use default value? I don't like adding one more argument to this function.